diff --git a/contrib/macos/dosbox-x.plist.in b/contrib/macos/dosbox-x.plist.in
index 3036455..fd3adcb 100644
--- a/contrib/macos/dosbox-x.plist.in
+++ b/contrib/macos/dosbox-x.plist.in
@@ -33,5 +33,9 @@
     <string>Copyright 2022 DOSBox-X</string>
     <key>LSMinimumSystemVersion</key>
     <string>10.8</string>
+    <key>CFBundleDisplayName</key>
+    <string>Prince Of Persia</string>
+    <key>CFBundleName</key>
+    <string>Prince Of Persia</string>
 </dict>
 </plist>
